src/components/Postform.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+
+class Postsform extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: '',
+            body: ''
+        };
+        this.onChange = this.onChange.bind(this);
+        this.onSubmit = this.onSubmit.bind(this);
+    }
+
+    onChange(event) {
+        this.setState({ [event.target.name]: event.target.value});
+    }
+
+    onSubmit(event) {
+        event.preventDefault();
+        const post = {
+            title: this.state.title,
+            body: this.state.body
+        };
+
+        fetch('https://jsonplaceholder.typicode.com/posts', {
```

Comment:
[jsonplaceholder](https://jsonplaceholder.typicode.com/) will give you the ability to fetch and HTTP request and it will provide dummy response. 
even you can add objects and the same can be expected in the response. This will help to set up fake response from the server.